### PR TITLE
Add events app with wagtail models

### PR DESCRIPTION
**Pull request by staaas**
This doesn't have UI yet, however it will be enough to create programs in advance.

The models were modeled after existing events app in the old website.

### CODE REVIEW

File: cpm_data/models.py [hunk 1]
```diff
@@ -78,6 +89,11 @@ class Film(ClusterableModel):
         related_name='+'
     )
 
+    objects = SearchableManager()
+
+    def __unicode__(self):
```
Comment (davojta) [inline cpm_data/models.py:94]: Why do we need that here? May be possible to add some comments to clear it out?

**Review by davojta — APPROVED** [review]
May be need to provide comment in PR or code

File: cpm_data/models.py (hunk 1 above)
Comment (staaas) [reply cpm_data/models.py:94]: that's actually a standard way to make human-readable object representations in Python

e397107b2 commit message contains the following:

> Add method __unicode__ to model cpm_data.Film
>
> So it is properly displayed by GenericModelChooserPanel in events app.

do you still want me to add a comment to the code? if yes, I can do it.

Comment (davojta) [reply cpm_data/models.py:94]: That info is enough)

**Pull request merged**